controllers/config/defaults.go

```diff
@@ -85,21 +85,22 @@ const (
 
 // DSPO Config File Paths
 const (
-	APIServerImagePath                       = "Images.ApiServer"
-	APIServerArtifactImagePath               = "Images.Artifact"
-	PersistenceAgentImagePath                = "Images.PersistentAgent"
-	ScheduledWorkflowImagePath               = "Images.ScheduledWorkflow"
-	APIServerCacheImagePath                  = "Images.Cache"
-	APIServerMoveResultsImagePath            = "Images.MoveResultsImage"
-	MariaDBImagePath                         = "Images.MariaDB"
-	OAuthProxyImagePath                      = "Images.OAuthProxy"
-	MlmdEnvoyImagePath                       = "Images.MlmdEnvoy"
-	MlmdGRPCImagePath                        = "Images.MlmdGRPC"
-	MlmdWriterImagePath                      = "Images.MlmdWriter"
-	ObjStoreConnectionTimeoutConfigName      = "DSPO.HealthCheck.ObjectStore.ConnectionTimeout"
-	DBConnectionTimeoutConfigName            = "DSPO.HealthCheck.Database.ConnectionTimeout"
-	RequeueTimeConfigName                    = "DSPO.RequeueTime"
-	ApiServerIncludeOwnerReferenceConfigName = "DSPO.ApiServer.IncludeOwnerReference"
+	APIServerImagePath                         = "Images.ApiServer"
+	APIServerArtifactImagePath                 = "Images.Artifact"
+	PersistenceAgentImagePath                  = "Images.PersistentAgent"
+	ScheduledWorkflowImagePath                 = "Images.ScheduledWorkflow"
+	APIServerCacheImagePath                    = "Images.Cache"
+	APIServerMoveResultsImagePath              = "Images.MoveResultsImage"
+	MariaDBImagePath                           = "Images.MariaDB"
+	OAuthProxyImagePath                        = "Images.OAuthProxy"
+	MlmdEnvoyImagePath                         = "Images.MlmdEnvoy"
+	MlmdGRPCImagePath                          = "Images.MlmdGRPC"
+	MlmdWriterImagePath                        = "Images.MlmdWriter"
+	ObjStoreConnectionTimeoutConfigName        = "DSPO.HealthCheck.ObjectStore.ConnectionTimeout"
+	DBConnectionTimeoutConfigName              = "DSPO.HealthCheck.Database.ConnectionTimeout"
+	RequeueTimeConfigName                      = "DSPO.RequeueTime"
+	ApiServerIncludeOwnerReferenceConfigName   = "DSPO.ApiServer.IncludeOwnerReference"
+	MlmdIgnoreAutoGeneratedSecretNotFoundError = "DSPO.Mlmd.IgnoreAutoGeneratedSecretNotFoundError"
```

Comment:
We'll ignore the secret on tests. Since tests don't run on OpenShift, the secret is never created.
Should I add a comment?

Reply:
We have tests that have TLS enabled, for example https://github.com/opendatahub-io/data-science-pipelines-operator/blob/cf1bd60823103ca99d4085e5f9c0a4c4e3fc3384/controllers/testdata/declarative/case_6/deploy/04_cr.yaml#L15

Reply:
let's disable it then, and create an action item jira for adding integration tests for this, it will be blocked by https://issues.redhat.com/browse/RHOAIENG-11646 is done

Reply:
Test removed and Jira created: https://issues.redhat.com/browse/RHOAIENG-13123.